portfolio/src/main/webapp/index.html

```diff
@@ -28,6 +28,19 @@ <h1>Messages</h1>
       <p>Click here to see messages from the server:</p>
       <button onclick="getMessages()">Fetch server messages!</button>
       <div id="messages-container"></div>
+
+      <form action="/data" method="POST">
+        <p>Enter a comment:</p>
+        <textarea name="comment-input">I love your website! :)</textarea>
```

Comment:
Consider using a placeholder instead of pre-populating content https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea#attr-placeholder

It'll give an example of the sort of thing people should type, but will automatically disappear when they start typing so they don't have to delete it first.

I mean, unless you only want this particular comment. I respect that :)